package.json

```diff
@@ -97,7 +103,7 @@
   ],
   "license": "Apache-2.0",
   "nyc": {
-    "report-dir": "coverage",
+    "report-dir": "cypress-coverage",
```

Comment:
@vickywane what is the purpose of setting nyc's report-dir parameter to the cypress-coverage dir?

Reply:
Initially i wanted to create seperate folders for the two reports. I have restored it back to the `coverage` name.